cosmo_tester/test_suites/test_security/secured_nodecellar_test.py

```diff
@@ -0,0 +1,113 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+from path import path
+
+from cosmo_tester.framework import util
+from cosmo_tester.framework.cfy_helper import CfyHelper
+from cosmo_tester.test_suites.test_blueprints.nodecellar_test \
+    import OpenStackNodeCellarTestBase
+
+from cloudify_rest_client.client import CloudifyClient
```

Comment:
the `cloudify_rest_client` and `cloudify_cli` imports should be the second part (between the standard lib imports the the system tests imports), because even though they are cloudify packages, they are 3rd party with regards to the cloudify-system-tests project